src/forms/FormFieldMapCoordinates/MapCoordinates.styled.tsx

```diff
@@ -11,19 +11,22 @@ export const StyledSpan = styled.span`
   color: ${theme.colors.gray600};
   font-family: ${theme.fontFamily};
   font-size: 14px;
+
+  @media (max-width: ${theme.breakpoints.mobile}) {
+    width: calc(100vw - 40px);
+  };
 `;
 
 export const CoordinatesCard = styled.div`
   border: 2px solid ${theme.colors.gray200};
   border-radius: 4px;
   display: flex;
   font-family: ${theme.fontFamily};
-  ${(pr) =>
-		pr.hasAddress ? "max-height: 204px;" : "height: 174px;"}
+	height: 210px;

```

Comment:
At least in github there seems to be some extra tabbing in this property. Also, the figma file seems to be using 204px.